lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs

```diff
@@ -521,6 +526,14 @@ withNetworkLayer tr network np project k = do
                         }
             void $ LN.lightSync (MsgLightLayerLog >$< tr) lightSyncSource follower
 
+    syncProgress :: BF.ClientConfig -> SlotNo -> IO SyncProgress
+    syncProgress bfConfig s = runBFM bfConfig do
+        BF.Block {_blockSlot} <- BF.getLatestBlock
+        let latestSlot = maybe 0 BF.unSlot _blockSlot
+            currentSlot = fromIntegral (unSlotNo s)
+            percentage = (currentSlot * 100) % latestSlot
```

Comment:
```suggestion
            percentage = currentSlot % latestSlot
```

The name is a bit confusing — `Percentage` is actually a rational number between [0,1]. For example, the definition of `clipToPercentage` is

```
clipToPercentage :: Rational -> Percentage
clipToPercentage = Percentage . min 1 . max 0
```

Reply:
Good catch!